libkineto/src/Config.cpp

```diff
@@ -577,37 +578,45 @@ void Config::setReportPeriod(milliseconds msecs) {
 }
 
 void Config::printActivityProfilerConfig(std::ostream& s) const {
-  s << "  Log file: " << activitiesLogFile() << std::endl;
+  s << "  Log file: " << activitiesLogFile();
```

Comment:
newline was dropped though:
```suggestion
  s << "  Log file: " << activitiesLogFile() < "\n";
```
or
```suggestion
  fmt::print(s, "  Log file: {}\n", activitiesLogFile());